src/Validate.php

```diff
@@ -547,6 +548,29 @@ public static function notIn(Validator $validator, $data, $pattern, $rule, $para
         }
     }
 
+    /**
+     * contains-only:<value>(,<value>)*
```

Comment:
You forgot to update this part of the comment
```suggestion
     * contains:<value>(,<value>)*
```